content/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Publicizing or hiding organization membership
-intro: 'If you''d like to tell the world which organizations you belong to, you can display the avatars of the organizations on your profile.'
+intro: 'If you''d like to show the world which organizations you belong to, you can show your avatars of the organizations on your profile.'
```

Comment:
```suggestion
intro: 'If you''d like to show the world which organizations you belong to, you can show the avatars of these organizations on your profile.'
```

(I'm a bit unsure about the `you can show the avatars of these organizations` bit. Any suggestions?)